### PR TITLE
Changes on continue statements

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Remove unnecessary 'continue' function
- Change 'continue' so that it breaks out of the 'switch', as there is no need to compare against the rest.

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

File: modules/mod_menu/helper.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public static function getList(&$params)
 						case 'separator':
 						case 'heading':
 							// No further action needed.
-							continue;
+							continue 2;
```
Comment (frankmayer) [inline modules/mod_menu/helper.php]: Yes, it should!! Good catch! Misread something in the PHP Manual.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 114a19a148d215dcf8e5a714496e97de615ed466<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13199">issues.joomla.org/tracker/joomla-cms/13199</a>.</sub>

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on 114a19a148d215dcf8e5a714496e97de615ed466<br /><br />Test OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13199">issues.joomla.org/tracker/joomla-cms/13199</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13199">issues.joomla.org/tracker/joomla-cms/13199</a>.</sub>

**Comment by rdeutz** [comment]
@zero-24 isn't restore.php managed different ?

**Comment by frankmayer** [comment]
@rdeutz yes, it seems so. I reverted a change in this very file in another PR as well.
Reverted. Now we should be good to go.

**Comment by rdeutz** [comment]
yeah good to go, had the check twice but continue and break are the same in a switch, break is more common.

**Pull request merged**